### DISCUSSION TITLE
Vertical tabs

### DISCUSSION THREAD

**Discussion by PoojMenon** [twilio-labs/paste#1495] [Q&A] [answered]
I'm exploring some layout options and noticed in [Figma](https://www.figma.com/file/XBsuXuvbILfuTpfDon1Zb1/2021-Q2-Revamp-Usage-Review-flows?node-id=203%3A70) that I can switch the Tab type to be 'vertical'. 

However, I couldn't find anything in the [documentation](https://paste.twilio.design/components/tabs/#paste-docs-content-area) regarding the same or on the roadmap. 

Am I missing something?

**Comment by TheSisb** [comment]
Hello @PoojMenon,

That's a good catch, I'll work on adding a vertical tabs example to our docs shortly.  In the meantime, you can view an [example on our Storybook](https://5e53448165911c0022e68c74-bvyleulyyc.chromatic.com/?path=/story/components-tabs--vertical-tabs).
I will report back when the docs are updated.

**Reply by PoojMenon** [reply]
Oh that's awesome! Thanks very much for the quick response Shadi.

**Comment by SiTaggart** [comment, marked as answer]
A minimal example of vertical tabs now live on the doc site https://paste.twilio.design/components/tabs#vertical-tabs

**Reply by PoojMenon** [reply]
Thanks Simon!
